docs/core/compatibility/2.2-3.0.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: Breaking changes, version 2.2 to 3.0 - .NET Core
```

Comment:
There should also be an *aspnet.md* or *aspnetcore.md* file that lists the .NET Core breaking changes. Right now, the TOC by category link for ASP.NET Core goes to GitHub; it should go to the aspnet.md page.